styleguide.css

```diff
@@ -246,106 +252,155 @@ input::placeholder {
 input:focus::placeholder {
     color:#bde513;
 }
-input[type=submit] {
+input[type=submit],
+button {
     text-align: center;
     padding: 2rem 4rem;
     cursor: pointer;
-}
-input[type=submit]:hover {
     background-color: #00283C;
     color: #fff;
 }
+input[type=submit]:hover,
+button:hover {
+    background-color: #fff;
+    color:  #00283C;
+}
 input[type=submit],
-input[type=submit]:hover {
+input[type=submit]:hover,
```

Comment:
Why do you need to specify the `:hover` here or on the button?

Reply:
in order to show the user it's a clickable button or input. Doesn't make sense ?